src/DocumentFormat.OpenXml.Linq/Feature/IPartRootXElementFeature.cs

```diff
@@ -11,6 +11,11 @@ namespace DocumentFormat.OpenXml.Packaging
     /// </summary>
     public interface IPartRootXElementFeature
```

Comment:
Maybe we should change this to `IPartRootXLinqFeature`

Reply:
Once we release it, adding members to an interface is breaking, so my thoughts with these have been to keep them fairly scoped and then add additional interfaces for new features. 

Yes, the main goal is to make it more generic mostly because now it's not *just* the XElement, but the XDocument as well.

Reply:
Can we make the interface internal?